weave

```diff
@@ -788,7 +788,9 @@ ipam_cidrs() {
     IPAM_CIDRs=""
     if [ $# -eq 0 ] ; then
         # If no addresses passed in, get one in the default subnet
-        IPAM_CIDRS=$(call_weave POST /ip/$CONTAINER_ID)
+        if ! IPAM_CIDRS=$(call_weave POST /ip/$CONTAINER_ID) ; then
```

Comment:
No; in the absence of anyone looking at the return value from `call_weave` its return value is overwritten by the next command, i.e. `[`.
This doesn't matter when `set -e` is in effect, but my next change calls `call_cidr` in an if-statement so removing the instant-exit behaviour.